catalog/spatial/csw/spatial-csw-source/pom.xml

```diff
@@ -151,6 +151,11 @@
             <artifactId>platform-util-unavailableurls</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>${commons-lang3.version}</version>
```

Comment:
you can just remove the version completely since it is in dependency management.